tests/dummy/app/pods/error/controller.js

```diff
@@ -0,0 +1,16 @@
+import Ember from 'ember'
+const {Controller} = Ember
+
+export default Controller.extend({
+  // BEGIN-SNIPPET error-query-params
+  queryParams: {
+    showDetails: {
+      replace: true
+    }
+  },
+  showDetails: false,
```

Comment:
Noted above, but I don't believe it's necessary to maintain this state externally - I'm open to someone telling me I'm wrong though :)